config.js

```diff
@@ -17,6 +17,7 @@ module.exports = {
       , 'es6/Reflect': 'bower_components/harmony-reflect/reflect'
       , 'codemirror': 'bower_components/codemirror'
       , 'jszip': 'bower_components/jszip/dist/jszip'
+      , 'filesaver': 'bower_components/file-saver.js/FileSaver'
```

Comment:
Did you try to load this in NodeJS?

For me:

```
$ metapolator dev-js-shell
metapolator> require('filesaver')
Error: Evaluating /home/commander/Projekte/metapolator/vectorworks/app/lib/bower_components/file-saver.js/FileSaver.js as module "filesaver" failed with error: TypeError: Cannot read property 'document' of undefined
    at Function.req.load (/home/commander/Projekte/metapolator/vectorworks/node_modules/requirejs/bin/r.js:2529:23)
    at Function.req.get (/home/commander/Projekte/metapolator/vectorworks/node_modules/requirejs/bin/r.js:2479:29)
    at Object.localRequire [as require] (/home/commander/Projekte/metapolator/vectorworks/node_modules/requirejs/bin/r.js:1650:40)
    at requirejs (/home/commander/Projekte/metapolator/vectorworks/node_modules/requirejs/bin/r.js:2003:24)
    at repl:1:2
    at REPLServer.self.eval (repl.js:112:21)
    at Interface.<anonymous> (repl.js:239:12)
    at Interface.EventEmitter.emit (events.js:95:17)
    at Interface._onLine (readline.js:202:10)
    at Interface._line (readline.js:531:8)
```

Because this is not compatible with nodeJS. `document` is part of the Web API.